package.json

```diff
@@ -3,7 +3,10 @@
   "private": true,
   "description": "",
   "scripts": {
+    "write-creds": "node ./scripts/write-credentials",
+    "prestart": "yarn write-creds",
     "start": "next start",
+    "predev": "dotenv yarn write-creds",
```

Comment:
Any command with `pre` prefixed to it will run before it's postfix (in this case dev). When we're running dev we want to read from the .env file, but when we're starting the service it'll come from an actual env var file.